src/components/GrampsjsFormEditMapLayer.js

```diff
@@ -229,20 +229,20 @@ class GrampsjsFormEditMapLayer extends GrampsjsObjectForm {
   }
 
   _handleMapClick(e) {
-    const map = this.shadowRoot.querySelector('grampsjs-map')
-    if (map !== null) {
-      const latlng = map._map.mouseEventToLatLng(e)
+    console.log('e', e)
```

Comment:
Can you please remove the log statement? Thanks!

Reply:
Resolved